### DISCUSSION TITLE
为什么将文件上传到github但访问博客网站却还是404

### DISCUSSION THREAD

**Discussion by Xiaoshen-QwQ** [hexojs/hexo#5566] [Question / Answer]
![捕获](https://github.com/user-attachments/assets/ab26cf89-d732-414a-8b01-e78b83ddfeee)
![捕获](https://github.com/user-attachments/assets/90c4c146-95d4-40ee-9bc5-c906c15d7912)

**Comment by Xiaoshen-QwQ** [comment]
以上问题已经解决，但还有一个问题，就是为什么markdown没有进行渲染？
![捕获](https://github.com/user-attachments/assets/69a3248e-d1a1-45be-92e7-2bcdb1f460f7)

**Reply by uiolee** [reply]
markdown has been rendered normally. Mathematical formulas are not part of markdown.

**Comment by uiolee** [comment]
Most likely you forgot to turn on the github page
